stores/metadata.go

```diff
@@ -1210,6 +1210,11 @@ func (s *SQLStore) ObjectEntries(ctx context.Context, bucket, path, prefix, sort
 		offset = 0
 	}
 
+	indexHint := ""
+	if !isSQLite(s.db) {
+		indexHint = "USE INDEX (idx_object_bucket, idx_objects_created_at)"
```

Comment:
Maybe we can add these to the test I added here <https://github.com/SiaFoundation/renterd/pull/1059>? 
If we end up closing out 1059 at least the test 'll survive. It's worth explicitly testing this imo.

Reply:
I'm not a huge fan of that test tbh. It simplifies the query we are actually running and runs it on a dataset that is not even close to a production dataset and only verify that we very able to force an index being used for that particular setup which might be detrimental. 
It's a bit like unit testing a mock. I think in this particular case it's more useful to run the exact query that was slow on the exact dataset it happened on and verify that the hint is speeding things up. If not, I'd avoid messing with the query planner.